src/assets/sass/protocol/base/elements/_quotes.scss

```diff
@@ -29,3 +37,8 @@ blockquote {
         @include font-mozilla;
     }
 }
+
+[dir='rtl'] {
```

Comment:
Testing the PR, I don't think you need this. The expected behaviour is to have the line on the right hand side of the citation for RTL languages, and that is where it appears for me without this.